src/Transports/MassTransit.Azure.ServiceBus.Core/AzureServiceBusTransport/Topology/SubscriptionEndpointSettings.cs

```diff
@@ -36,6 +36,7 @@ public SubscriptionEndpointSettings(IServiceBusEndpointConfiguration configurati
         public IServiceBusSubscriptionConfigurator SubscriptionConfigurator => _subscriptionConfigurator;
 
         public override bool RequiresSession => _subscriptionConfigurator.RequiresSession ?? false;
+        public override int MaxConcurrentSessions => _subscriptionConfigurator.MaxConcurrentSessions ?? 1;
```

Comment:
Is the the SDK default value? If not, it should default to the SDK default value.

Reply:
Yes, will move this and MaxConcurrentCallsPerSession to Defaults.

Of interest, MT's [`SessionIdleTimeout =TimeSpan.FromSeconds(10)`](https://github.com/MassTransit/MassTransit/blob/5cc00c25988cf9fdbe8e987f19d89bb2bf3b33b4/src/Transports/MassTransit.Azure.ServiceBus.Core/AzureServiceBusTransport/Defaults.cs#L26) is not the default.
* SDK has it as null (a TimeSpan?).  [When not specified](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebussessionprocessoroptions?view=azure-dotnet#properties), SDK will use [`TryTimeout`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusretryoptions.trytimeout?view=azure-dotnet#azure-messaging-servicebus-servicebusretryoptions-trytimeout) which defaults to 60s (not overridden by MT)

Would you like `SessionIdleTimeout ` made consistent with the SDK?    TimeSpan?=null

Reply:
Sure, that would be great.